added doc on eformidling contract (#180)
Altinn/altinn-studio-docs: content/teknologi/altinnstudio/altinn-api/models/app-metadata/_index.md

dataTypeId | The name of the datatype. See [DataType](#datatype). 

## eFormidlingContract

This type is used by a feature still in development. 

eFormidlingContract holds the configuration of the eFormidling integration for the application. 

> An application configured to enable eFormidling integration in combination with the eFormidiling contract will send a shipment to eFormidling for each instance that is created of the application. 

Reviewer: ```suggestion
An application configured to enable eFormidling integration in combination with the eFormidiling contract will send a shipment to eFormidling for every instance that is created. 
```
